source/manual/govuk-env-sync.html.md

```diff
@@ -18,12 +18,12 @@ In this context:
 
 The govuk_env_sync data sync works by `push`ing a source database to S3 and subsequently `pull`ing it down from there to a destination. There is a different process for the Elasticsearch databases, which [don't push/pull](https://github.com/alphagov/govuk-puppet/blob/cb9351f92456ccf132e776b3b9f7129c0e654697/modules/govuk_env_sync/files/govuk_env_sync.sh#L496-L504) but instead get [copied to another Elasticsearch](https://github.com/alphagov/govuk-puppet/blob/cb9351f92456ccf132e776b3b9f7129c0e654697/modules/govuk_env_sync/files/govuk_env_sync.sh#L321-L337).
 
-The environment synchronisation is achieved by granting cross-account access of the `govuk-<environment>-database-backups` S3 buckets. Figure 1 below illustrates the intended data flow between environments.
+The environment synchronisation is achieved by granting cross-account access of the `govuk-<environment>-database-backups` S3 buckets.
 
-![Schematic of the data flow of the govuk_env_sync data synchronisation](images/govuk_env_sync.png)
-Figure 1:  Schematic of the data flow of the govuk_env_sync data synchronisation.
-
-In practice, some Integration databases pull from Production (with data sanitisation - see below) rather than from Staging. This is done to reduce the number of cron job timing dependencies and to save resources where large databases do not need to be backed up from Staging.
+* In production, databases _push_ to the `govuk-production-database-backups` S3 bucket.
+* In staging, databases are replaced by _pulling_ data from `govuk-production-database-backups`.
+* In integration, databases are replaced by _pulling_ data from `govuk-production-database-backups` - some databases have a [data sanitisation process](#data-sanitisation).
+* In integration, databases are _pushed_ to `govuk-integration-database-backups` nightly - these are produced for developers to use [real data on their local machines](/repos/govuk-docker/how-tos.html#how-to-replicate-data-locally).
```

Comment:
Nice one for explaining why integration databases are backed up, as I know this came up a couple of times in the recent DB upgrade work.

Is it worth explicitly stating that staging doesn't have a mechanism for pushing its data to a `govuk-staging-database-backups` bucket (as implied in its absence above)?

Reply:
I don't think I really want to dig into that too much as it ends up a bit confusing with the special case for Whitehall. I feel like I'll either leave a lie or it be overly verbose.